clouddriver-openstack/src/main/groovy/com/netflix/spinnaker/clouddriver/openstack/deploy/ops/servergroup/DeployOpenstackAtomicOperation.groovy

```diff
@@ -177,4 +184,20 @@ class DeployOpenstackAtomicOperation implements AtomicOperation<DeploymentResult
     result
   }
 
+  /**
+   * Return the file contents of a template, either from the account config location or from the classpath.
+   * @param filename
+   * @return
+   */
```

Comment:
We could cache in simple map. Loading the template is going to be a small fraction of the time it takes to create the server group, though.